schedule/yam/agama_interactive.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: agama_interactive
+description: >
+  Perform nteractive installation with agama.
+schedule:
+  - installation/bootloader_start
+  - yam/agama/dummy
```

Comment:
let's call it `agama_is_visible.pm` but whatever you like more... :)

Reply:
Ok, NP for me.